Use ES6 variable declarations in expressions related examples

ikarasz opened the pull request:
This pull request aims to replace the `var` keword with the proper ES6 decalartions in the the expressions related examples

wbamberg commented on live-examples/js-examples/expressions/expressions-propertyaccessors.html:
```diff
@@ -1,14 +1,17 @@
 <pre>
-<code id="static-js">var person = {};
+<code id="static-js">const person1 = {};
 person['firstname'] = 'Mario';
```
```suggestion
person1['firstname'] = 'Mario';
```

wbamberg commented on live-examples/js-examples/expressions/expressions-propertyaccessors.html:
```diff
@@ -1,14 +1,17 @@
-<code id="static-js">var person = {};
+<code id="static-js">const person1 = {};
 person['firstname'] = 'Mario';
 person['lastname'] = 'Rossi';
```
```suggestion
person1['lastname'] = 'Rossi';
```

ikarasz commented:
Thanks for your review @wbamberg, I fixed the issue and also adjusted the expected output in one [example](https://github.com/mdn/interactive-examples/pull/1499/commits/984e971dabc3533d5f73ec365ae8753db7e212e4)
